app.py

```
@@ -0,0 +1,135 @@
+@asynccontextmanager
+async def lifespan(app: FastAPI):
+    try:
+        ca = certifi.where()
+        MONGO_DB_URL = os.getenv("MONGO_DB_URI")
+        client = pymongo.MongoClient(MONGO_DB_URL, tlsCAFile=ca)
+        app.state.db_client = client
+        app.state.db = client[DATA_INGESTION_DB]
+        app.state.collection = app.state.db[DATA_INGESTION_COLLECTION]
+        logger.info("✅ MongoDB connection established.")
+        yield
+    except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)